tests/system/Commands/Database/ShowTableInfoTest.php

```diff
@@ -67,12 +67,8 @@ public function testDbTable(): void
         $expected = 'Data of Table "db_migrations":';
         $this->assertStringContainsString($expected, $result);
 
-        $expected = <<<'EOL'
-            +----+----------------+--------------------+-------+---------------+------------+-------+
-            | id | version        | class              | group | namespace     | time       | batch |
-            +----+----------------+--------------------+-------+---------------+------------+-------+
-            EOL;
-        $this->assertStringContainsString($expected, $result);
+        $expectedPattern = '/\|\hid\h+\|\hversion\h+\|\hclass\h+\|\hgroup\h+\|\hnamespace\h+\|\htime\h+\|\hbatch\h\|/';
```

Comment:
This is difficult for humans to read.

Is this better?
```suggestion
        $expectedPattern = '/\| id[[:blank:]]+\| version[[:blank:]]+\| class[[:blank:]]+\| group[[:blank:]]+\| namespace[[:blank:]]+\| time[[:blank:]]+\| batch \|/';
```